src/main/java/willy/task/ToDoTask.java

```diff
@@ -0,0 +1,16 @@
+package willy.task;
+
+/**
+ * Basic Task with no additional conditions.
+ */
+public class ToDoTask extends Task {
```

Comment:
Perhaps the class name should be consistent with the "DeadlineTask", like "TodoTask"?